models/configuration.yaml

```diff
@@ -798,6 +798,9 @@ server:
     send-proxy-v2:
       type: string
       enum: [enabled, disabled]
+    slowstart:
+      type: string
+      pattern: '^[1-9][0-9]*(us|ms|s|m|h|d)$'
```

Comment:
This is correct given the docs, but in the API we handle all time values as `ms` which is default for haproxy, it's just easier to work in it with API, so when we read string values with suffix from, we report it as equivalent `integer` value in ms in API responses. When writing it back to the config we always write it as integer in ms.

So here `slowstart` should be `type: integer`. Also can you add this parameter to the `default_server` key in `defaults` and `backed` object to be consistent.